tcms/testruns/static/testruns/js/get.js

```diff
@@ -267,17 +264,18 @@ function changeAssigneeBulk() {
     });
 }
 
-function updateExecutionText() {
-    const executions = 0 // todo: this is the list of all selected executions
-    if (!executions.length) {
-        window.alert(default_messages.alert.no_case_selected);
-        return false;
+function updateCaseText() {
+    const selected = selectedCheckboxes()
+    if ($.isEmptyObject(selected)) {
+        return false
     }
-    //todo: translations
-    executions.forEach(executionId =>
-        jsonRPC('TestExecution.update', [executionId, { 'case_text_version': 'latest' }], () => { }, sync = true)
+
+    selected.executionIds.forEach(executionId =>
+        jsonRPC('TestExecution.update', [executionId, { 'case_text_version': 'latest' }], () => {
+            // TODO: ATM we don't show this info anywhere
+            // once we do update the HTML element here
```

Comment:
With the new `reloadRowFor()` will be easier.

Reply:
I need the whole `execution` to call `reloadRowFor`, not only the id. ATM I don't have it here and don't have a sensible way to get it. Once we keep the state in the js, I will be able to get it from there via the id, but not now. let's keep this as draft and defer if for later.

Reply:
oh, wait `TestExecution.update` returns the updated execution.